source/common/access_log/access_log_formatter.cc

```diff
@@ -377,6 +377,11 @@ StreamInfoFormatter::StreamInfoFormatter(const std::string& field_name) {
         return UnspecifiedValueString;
       }
     };
+  } else if (field_name == "ROUTE_NAME") {
```

Comment:
Please add this change to the public docs as well as version history. Also, please make a parallel change to add this to the gRPC access logs.